sdk/src/types/block/output/delegation.rs

```diff
@@ -41,13 +41,19 @@ impl DelegationId {
     }
 }
 
+#[derive(Copy, Clone)]
+pub enum DelegatedAmount {
```

Comment:
I'm not too convinced by this but I also think we should reverse amount/delegated amount and we have an issue for this so I'll ignore until we address it.